build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
     implementation 'org.springframework:spring-context-support'
     implementation 'com.google.guava:guava:28.2-jre'
     compileOnly 'org.projectlombok:lombok'
+    compile group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.64'
```

Comment:
Follow same convention as other imports.

Reply:
and compile is deprecated in gradle 7, replace compile with Implementation.

Reply:
Done